src/types/Todo.ts

```diff
@@ -0,0 +1,6 @@
+export interface Todo {
+  id?: number;
```

Comment:
Should everything be optional here? - I don't think so. You can create a correct interface and use Partial, Omit, Pick utility types where you need